src/Infrastructure/Persistence/Specification/Evaluators/FSHSpecificationEvaluator.cs

```diff
@@ -0,0 +1,26 @@
+using Ardalis.Specification;
+using Ardalis.Specification.EntityFrameworkCore;
+
+namespace FSH.WebApi.Infrastructure.Persistence.Specification.Evaluators;
+
+public sealed class FSHSpecificationEvaluator : SpecificationEvaluator
+{
+    public static FSHSpecificationEvaluator DefaultInstance { get; } = new FSHSpecificationEvaluator();
+    public static FSHSpecificationEvaluator NpgsqlInstance { get; } = new FSHSpecificationEvaluator(true);
```

Comment:
I'm not sure how inefficient is to keep both singleton instances in memory if only one of them will be ever used but this is how in Ardalis.Specification Default and Cached instances are kept so I did the same (https://github.com/ardalis/Specification/blob/b64cb0144d4dcda0fc5a9d927345cf3ceb5e076b/Specification.EntityFrameworkCore/src/Ardalis.Specification.EntityFrameworkCore/Evaluators/SpecificationEvaluator.cs#L19)

Reply:
Yeah, I saw that in the specification source as well. I think it's ok like this.